rust/worker/src/execution/operators/prefetch.rs

```diff
@@ -0,0 +1,216 @@
+use thiserror::Error;
```

Comment:
Should we somehow tag this as I/O bound and then let the dispatcher run it on the main runtime?

Reply:
Yes the next PR